script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -428,16 +434,17 @@ But if you're trying to produce vampires of Rank II and above,
 // Rendering 特性 as 'powers' here since, as I interpret this,
 // "it is a trait of vampires that they can control the Dead who have been injected with their vampire blood"
 // but that's a bit verbose and doesn't lead as nicely into the next line.
+// not 'move', 'work'. not quite the same thing.
 "Most humans don't have this aptitude.
- As a result, once they've been injected with a vampire's blood,
- that vampire can use their own powers to control them.
- It's like a disease that makes you move even after death.
+ As a result, once they've been injected with vampire blood,
```

Comment:
"blood" can take species before it like this:

human blood
dolphin blood
monkey blood
elephant blood
cat blood

These are normal expressions. Sometimes it is phrased as "cat's blood" instead of "cat blood" but this is less common. Having the attributive form (feline blood) of a noun used instead is similarly uncommon.